src/common/icon/default-icons.ts

```diff
@@ -1,6 +1,6 @@
-import { Icon } from "./icon";
+import {Icon} from "./icon";
 
-import { IconType } from "./icon-type";
+import {IconType} from "./icon-type";
```

Comment:
These changes are unnecessary.

Reply:
This changes were made by WebStorm when formatting the file.
Should I change it back or keep it?

Reply:
Change it back.